starters/basic-starter/components/meta.tsx

```diff
@@ -0,0 +1,95 @@
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { DrupalMetatag } from 'types/drupal';
+
+interface MetaProps {
+  title?: string;
+  path?: string;
+  tags?: DrupalMetatag[];
+}
+
+export function Meta({ tags }: MetaProps) {
+  const router = useRouter();
+  const origin =
+    typeof window !== 'undefined' && window.location.origin
+      ? window.location.origin
+      : '';
+  const currentUrl = `${origin}${router.asPath !== '/' ? router.asPath : ''}`;
+
+  // Initialize imageUrl variable with default value.
+  let imageUrl = `${origin}/_next/image?url=`;
+  // Check whether the tags is an array list.
+  if (Array.isArray(tags)) {
+    // Filter out image property from og:image
+    const filterImage = tags.filter((obj) => {
+      return obj.attributes.property === 'og:image';
+    });
+    // Prepare image url.
+    const imageSrc = filterImage[0].attributes.content;
```

Comment:
@rajeshreeputra Image is optional field in Event & Article content type. So, we shouldn't assume that `og:image` tag will always exists.Can we add condition check ?